.github/workflows/packer-build-and-publish.yml

```diff
@@ -8,22 +8,30 @@ permissions:
   id-token: write
 
 jobs:
-  get-branch:
+  check-if-main:
     runs-on: ubuntu-latest
     outputs:
-      branch: ${{ steps.branch.outputs.value }}
+      is-main: ${{ steps.is_main.outputs.value }}
     steps:
-      - name: Checkout repository
+      - name: Error on non-tag events
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+        run: |
+          echo "This workflow can only be triggered by tag events."
+          exit 1
```

Comment:
Can this be done with an `if` on the job to skip the whole thing? Running a step starts a VM and all of that so it'd be good to skip the whole thing.

Reply:
Done!